sycl/test-e2e/Plugin/level_zero_sub_sub_device.cpp

```diff
@@ -91,6 +123,12 @@ int main(void) {
 
   device d;
 
+  // PVC-1T does not support partition by affinity domain
+  if (!IsPVC_MultiTiles(d)) {
+    std::cout << expected_output << std::endl; // trick FileCheck
+    return 0;
+  }
+
```

Comment:
I only have access to the a PVC-1T, but the IsPVC() test retrieves the device_id and verifies it with a mask. It is returning true for both PVC-1T and PVC-2T.  It's possible that they differ in the range of that mask, but I haven't verified that by hand.  But rather than key off of some poorly understood device id, I think it's better to do as we are here: actually check that the the AffinityDomain partitioning is available.  The test needs it, so its correct to check it.